nav2_msgs/action/FollowGPSWaypoints.action

```diff
@@ -0,0 +1,14 @@
+# Error codes
+# Note: The expected priority order of the errors should match the message order
+uint16 NONE=0
```

Comment:
Hi @SteveMacenski! I was updating this branch and I'm not sure if this error codes would hold for this action server as well